.github/scripts/update-translations/TranslationManager.py

```diff
@@ -0,0 +1,134 @@
+'''
+	Automatically applies translation based on the dev setup.
+	Translation that are removed there, will be removed in all the others.
+	Translation that are added there, will be add in all the others with the given text or the english text.
```

Comment:
what if a different text exists in the translation files than in the master translation file?